routes/webhook.js

```diff
@@ -28,16 +28,14 @@ async function handleGitHubCall(request, response) {
   const pr = body.pull_request
   try {
     switch (body.action) {
-      case 'opened': {
-        await curationService.prOpened(pr)
+      case 'opened':
+      case 'synchronize': {
+        curationService.validateContributions(pr.number, pr.head.sha)
```

Comment:
little hard to tell but it looks like this will `_getPrFiles` twice as both validate and update call `_getContributedCurations` which goes to GitHub and gets the files. truth?

Reply:
nice find!

I'll push an update